### DISCUSSION TITLE
Question Regarding the Shifting Coefficient in DualSPHysics Flapping Flexible Beam Example

### DISCUSSION THREAD

**Discussion by JxZhou21** [DualSPHysics/DualSPHysics#239] [DUALSPHYSICS_V5.2]
Dear DualSPHysics,

I am a beginner in using DualSPHysics, currently conducting simulations on the flapping flexible beam (Turek and Hron example) case provided in the example files. However, I am puzzled by the definition of the shifting coefficient, which is set to -20 in the example. This value seems different from the recommended values for A (between 2 and 6), as described in Equation 26 of the SPH formulation introduced by DualSPHysics (details can be found [here](https://github.com/DualSPHysics/DualSPHysics/wiki/3.-SPH-formulation)).

Could you please clarify the difference between the shift coefficient used in the example file and the recommended values in Equation 26? Additionally, would it be feasible to change this value to a positive one, such as 20? If so, what would be the implications or consequences of using a positive shift coefficient?

Thank you for your time and assistance.

Best regards,
Jiaxing

**Comment by joconnor22** [comment]
Hi, thanks for your query. While it is true that for most problems the recommended value for shifting is somewhere between -1 and -6. However, for this problem, where we have vortex shedding off the back of the cylinder, it is necessary to have a larger shifting coefficient to preserve a regular particle distribution and prevent large voids forming in the particle distribution.

Regarding changing the sign of the shifting coefficient - I have never tried this before. However, I expect this will have the inverse effect (i.e. instead of ensuring particles follow a regular distribution they will clump together). This is because particles will now move towards regions of a higher concentration gradient. Because of this, I don't recommend trying a positive shifting coefficient.

I hope this answers your question and if so please consider closing the issue. Thanks.
